catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/RetreiveAllQueryRequests.java

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.metacard;
+
+import static org.codice.ddf.catalog.ui.metacard.query.util.QueryAttributes.QUERY_TAG;
+
+import com.google.common.collect.ImmutableSet;
```

Comment:
Guava is awesome but the way they version is terrible so we should avoid it when we can use standard Java I think something like collections.unmodifiableset(new hashset(Arrays.aslist(...))) instead

Reply:
+1

Reply:
@ryeats What does guava do poorly in regards to versioning?

The readability of the standard java solution is so much worse...

```
Collections.unmodifiableSet(new HashSet(Arrays.asList(Core.TITLE, Core.METACARD_OWNER, Core.DESCRIPTION, QueryAttributes.QUERY_SOURCES)));
```

versus

```
ImmutableSet.of(Core.TITLE, Core.METACARD_OWNER, Core.DESCRIPTION, QueryAttributes.QUERY_SOURCES);
```